srcpkgs/cpputest/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'cpputest'
-pkgname="cpputest"
-version="3.8"
+pkgname=cpputest
+version=4.0
 revision=1
-build_style="cmake"
+build_style=cmake
 short_desc="CppUTest unit testing and mocking framework for C/C++"
 maintainer="Julien Dehos <dehos@lisic.univ-littoral.fr>"
+license="BSD-3-Clause"
 homepage="http://cpputest.github.io/"
-license="BSD"
-distfiles="https://github.com/cpputest/${pkgname}/releases/download/v${version}/${pkgname}-${version}.tar.gz"
-checksum="c81dccc5a1bfc7fc6511590c0a61def5f78e3fb19cb8e1f889d8d3395a476456"
+distfiles="https://github.com/cpputest/cpputest/releases/download/v${version}/cpputest-${version}.tar.gz"
+checksum=21c692105db15299b5529af81a11a7ad80397f92c122bd7bf1e4a4b0e85654f7
 configure_args+=" -DTESTS=OFF"
```

Comment:
either move this below `build_style` or make it conditional on `xXBPS_CHECK_PKGS=yes`. if it won't work, comment about the reason

Reply:
Moved line and rebases onto master. Thank you for the review.

Reply:
```
Linting srcpkgs/cpputest/template...

srcpkgs/cpputest/template: Place configure_args+= after checksum=
```

Reply:
make it conditional on `XBPS_CHECK_PKGS`, some people like to run tests

Reply:
Tests work only on native builds. Is disabling them on cross OΚ?

Reply:
of course tests only work on native builds. that is implied , no need check on $CROSS_BUILD.

Reply:
see `srcpkgs/json-c++/template` for an example, set `XBPS_CHECK_PKGS=yes` in `etc/conf` to run do_check

Reply:
> of course tests only work on native builds. that is implied , no need check on $CROSS_BUILD.

It runs tests during build.

Reply:
The problem was not actual with building or running tests but rather with discovering tests during build. I disabled it on cross and tested locally with aarch64 and armv7l.